app/angular/AngularView/src/app/home/home.component.html

```diff
@@ -1,18 +1,126 @@
 <!DOCTYPE html>
+<title>Home</title>
 <body>
-    <h1>Inspect . Book . Get</h1>
-    <h2>"Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit..."</h2>
     <div class="container">
-        <div class="user-button">
-            <div class="img"></div>
-            <a id="but-1" mat-button mat-stroked-button routerLink="../items" routerLinkActivate = "active">
-                Check Item List
-            </a>
-            <a id="but-2" mat-button mat-stroked-button routerLink="../edit" routerLinkActivate = "active">
-                Make a Booking
-            </a>
+        <!-- section 1 of the homepage: The main contents and buttons -->
+        <div class="sec1">
+            <div class="main">
+                <div class="left">
+                    <div class="main_caption">
+                        <h1>Booking has never been so <span id="EASY">EASY</span></h1>
+                        <p>We, commIT technical cell strive to cater you the best experience 
```

Comment:
this text needs to be changed, but that's an issue for later

Reply:
Text changed. Please check if it is appropriate. Thank you.

Reply:
All the additional files are moved into assets file.